Content.Shared/CCVar/CCVars.cs

```diff
@@ -22,104 +22,6 @@ public sealed partial class CCVars : CVars
         CVarDef.Create("debug.option_visualizer_test", false, CVar.CLIENTONLY);
 
     /// DELTA-V CCVARS
-    /*
-     * Glimmer
```

Comment:
wtf happened here lol

Reply:
When we merged supermatter these cvars snuck in